src/components/dato/PeriodeDatovelger.tsx

```diff
@@ -15,13 +15,15 @@ interface Props {
   periode: IPeriode;
   settDato: (dato: Date | null, objektnøkkel: string) => void;
   showMonthYearPicker?: boolean;
+  datobegrensing?: any;
```

Comment:
Går det an å sette `  datobegrensning: DatoBegrensning ` i stedet for any?

Reply:
Yes, fikset :)